python/lsst/dax/imgserv/jobqueue/imageworker.py

```diff
@@ -78,17 +80,30 @@ def get_image_async(self, *args, **kwargs):
     config["DAX_IMG_CONFIG"] = config_path
     meta_url = imgserv_config.webserv_config["dax.imgserv.meta.url"]
     config["DAX_IMG_META_URL"] = meta_url
-    soda = imageSODA.ImageSODA(config)
-    image = soda.do_sync(params)
-    # save the result image in local temp dir
-    with tempfile.NamedTemporaryFile(dir=config["DAX_IMG_TEMPDIR"],
+    image = None; status = "OK"
```

Comment:
This line just looks weird.  If you want to have it as one line, I guess you could do something like (image, status) = (None, "OK") or something, but this just looks strange since you don't use this pattern anywhere else.

Reply:
Gone.